scripts/push-schema-changes.js

```diff
@@ -61,20 +62,19 @@ async function main() {
 
     execSync("yarn dump:staging")
 
-    await updateSchemaFile({
-      repo: "eigen",
-      body:
-        "Greetings human :robot: this PR was automatically created as part of metaphysics' deploy process. #nochangelog",
-    })
-    await updateSchemaFile({ repo: "force" })
-    await updateSchemaFile({
-      repo: "volt",
-      dest: "vendor/graphql/schema/metaphysics.json",
-    })
-    await updateSchemaFile({
-      repo: "pulse",
-      dest: "vendor/graphql/schema/metaphysics.json",
-    })
+    const updateOptionses = [
```

Comment:
Is `Optionses` a real word? Maybe `reposToUpdate = [` might be a bit clearer?

Reply:
That's for @pepopowitz he loves it when I name things like that! JK your suggestion is better - will update.